docs/cluster-large.md

```diff
@@ -0,0 +1,56 @@
+# Kubernetes Large Cluster
+
+## Support
+At v1.0, Kubernetes supports clusters up to 100 nodes with 30-50 pods per node and 1-2 container per pod (as defined in the [1.0 roadmap](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/roadmap.md#reliability-and-performance)).
```

Comment:
Is there a spec for vertical scale that k8s supports? Does k8s support 32 core nodes? @dchen1107